src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -159,40 +166,53 @@ private Task WriteBadRequestResponseAsync(HttpContext context, IDocumentWriter w
             return writer.WriteAsync(context.Response.Body, result);
         }
 
-        private Task WriteResponseAsync<TResult>(HttpContext context, IDocumentWriter writer, TResult result)
+        private async Task WriteResponseAsync<TResult>(HttpContext context, IDocumentWriter writer, TResult result)
```

Comment:
Why change the method signature if you can simply return the task?

Reply:
With the addition of Async/Await, the out parameters were not allowed.

Reply:
Sorry but I don't understand. What out parameters?